applications/FemToDemApplication/custom_elements/femdem3d_element.cpp

```diff
@@ -149,7 +149,7 @@ void FemDem3DElement::ComputeEdgeNeighbours(ProcessInfo &rCurrentProcessInfo)
 	Matrix nodes_indexes = ZeroMatrix(6, 2);
 	this->SetNodeIndexes(nodes_indexes);
 
-	// Loop over EDGES to assign the elements that share that edge -> Fill mEdgeNeighboursContainer
+	// Loop over EDGES to assign the elements that share that edge -> Fill mp_edge_neighboursContainer

```

Comment:
Member variables are CammelCase, it was fine

Reply:
done